client/state/themes/selectors.js

```diff
@@ -695,3 +696,14 @@ export function shouldFilterWpcomThemes( state, siteId ) {
 		! isJetpackSiteMultiSite( state, siteId )
 	);
 }
+
+/**
+ * Returns the URL for purchasing a plan for the current site.
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {Number}  siteId  Site ID for which to buy the theme
+ * @return {?String}         Plan purchase URL
+ */
+export function getPlanUpgradeUrl( state, siteId ) {
+	return '/plans/' + getSiteSlug( state, siteId );
```

Comment:
Sounds great. In this case there's only one plan the user should purchase so yeah, it's good.